libbeat/publisher/queue/memqueue/broker.go

```diff
@@ -380,10 +388,14 @@ func (b *batch) Count() int {
 	return len(b.entries)
 }
 
-func (b *batch) Event(i int) interface{} {
+func (b *batch) Entry(i int) interface{} {
 	return b.entries[i].event
 }
 
+func (b *batch) ID(i int) queue.EntryID {
+	return b.entries[i].id
+}
+
```

Comment:
Don't we need a boundary check and a returned error in exported functions like this? It should not panic if the `i` is out of range, I think?